### PR TITLE
Fix for integration tests by revert to Anorm 2.5.0

### PR DESCRIPTION
**Pull request by wsargent**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -10,7 +10,7 @@ libraryDependencies ++= Seq(
   jdbc,
   evolutions,
   "com.adrianhurt" %% "play-bootstrap" % "1.0-P25-B3",
-  "com.typesafe.play" %% "anorm" % "2.5.1",
+  "com.typesafe.play" %% "anorm" % "2.5.0",
```
Comment (cchantep) [inline build.sbt:13]: What is the issue there?

Comment (wsargent) [inline build.sbt:13]: I'd get exceptions like:

```
[error] application -

! @704nhlnc4 - Internal server error, for (GET) [/computers] ->

play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[AnormException: ColumnName(COMPUTER.NAME,Some(NAME))]]
    at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:280)
    at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:206)
    at play.api.GlobalSettings$class.onError(GlobalSettings.scala:160)
    at play.api.DefaultGlobal$.onError(GlobalSettings.scala:188)
    at play.api.http.GlobalSettingsHttpErrorHandler.onServerError(HttpErrorHandler.scala:98)
    at play.core.server.netty.PlayRequestHandler$$anonfun$2$$anonfun$apply$1.applyOrElse(PlayRequestHandler.scala:100)
    at play.core.server.netty.PlayRequestHandler$$anonfun$2$$anonfun$apply$1.applyOrElse(PlayRequestHandler.scala:99)
    at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:346)
    at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:345)
    at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
Caused by: anorm.AnormException: ColumnName(COMPUTER.NAME,Some(NAME))
[DEBUG] [05/15/2016 06:02:20.490] [application-akka.actor.default-dispatcher-4] [EventStream] shutting down: StandardOutLogger started
[info] application - Shutting down connection pool.
[info] c.z.h.p.HikariPool - HikariPool-2 - is closing down.
```

on 2.5.1

Comment (cchantep) [inline build.sbt:13]: @wsargent due to resolution priority when a simple name (there `NAME`) can be derived several qualified names in the same resultset (there `COMPUTER.NAME` and `COMPANY.NAME`).

See https://github.com/playframework/anorm/issues/102

Comment (wsargent) [inline build.sbt:13]: Okay, I will add a PR to fix that.

Comment (cchantep) [inline build.sbt:13]: It's fixed with Anorm master, unfortunately, the fix was not backport on the 2.5.x branch: https://github.com/playframework/anorm/pull/109

**Pull request merged**